### DISCUSSION TITLE
Calcule de produits issus de commerce équitable 2021

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1000] [Idées]
Dans 2021 le pourcentage 50 % de qualité et durables va inclure les produits issus de commerce equitable.

On va mettre à jour le calcul pour toutes les années (par exemple dans le graphe collonnes avec les données de toutes les années)

Comment s'assurer que les valeurs ne vont pas doubler entre commerce équitable et bio ?

**Comment by valeriemerle** [comment]
dans le tableur en téléchargement on a clairement indiqué une regle pour les produits ayant un double (parfois triple) etiquetage. Peut ere faut il le rappeler dans l'outil de suivi des achats. L'utilisateur doit choisir une seule catégorie quand le produit peut entrer dans plusieurs catgories. Dans ton exemple Helen, il faudrait que l'utilisateur choisisse entre bio et produits issus du "commerce equitable " sauf si on peut faire un double comptage dans la base de données mais qui n'a pas d'impact sur le calcul du total des achats?

**Reply by hfroot** [reply]
Si l'utilisateur utilise le suivi d'achats, il peut carrément mettre plusieurs étiquettes, et on peut se mettre d'accord sur comment on veut compter cet achat (par exemple, priorise bio) avec les règles automatiques. Par contre, oui je suis d'accord qu'on doit clarifier notre UX pour expliquer qu'il faut que les achats ne soient pas compter en doublons etc, particulierement pour les gens qui n'utilise pas l'outil suivi achats
